manuscript/index.qmd

```diff
@@ -78,19 +78,36 @@ We produce the simulations described in the next section for both of these GIs.
 
 ##### Reproduction number scenarios
 
-We test the following scenario:
-- Piecewise constant Rt
-    - 1.1 for two weeks
-    - 2 for two weeks
-    - 0.5 for two weeks
-    - 1.5 for two weeks
-    - 0.75 for two weeks
-    - 1.1 for six weeks
-    - sine curve centered at 1 with amplitude of 0.3 afterwards
+We test the following scenarios grouped by the **Outbreak**, **Endemic**, and **Miscellaneous** categories:
+
+**Outbreak scenarios**
+
+This is the list of scenarios where the initial number of infections is small but $R_t$ is initially significantly greater the 1 (e.g. $R_t > 1.5$).
+
+- _Susecptible depletion_. A smooth transition over time from $R_t > 1$ to $R_t < 0$. This represents a scenario where decrease in $R_t$ is due to greater population immunity, although it should be noted that we aren't modelling that effect mechanistically.
+- _Susecptible depletion with measures_. A sharp/discontinuous transition over time from $R_t > 1$ to $R_t < 0$, followed by a sharp transition back to $R_t > 1$ and then smooth transition to $R_t < 0$ again. This represents a scenario where initial decrease in $R_t$ is due to implementation of public health measures to reduce transmission. The sharp transition back to $R_t > 1$ is due to later relaxation of measures.
```

Comment:
I think we want the sharp transition to be prior to R_t hitting 0

Reply:
i.e If you mean Rt < 1 here that makes sense to me

Reply:
Yes. Typo